src/status_im2/contexts/onboarding/common/intro/view.cljs

```diff
@@ -0,0 +1,94 @@
+(ns status-im2.contexts.onboarding.common.intro.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [utils.re-frame :as rf]
+            [status-im2.contexts.onboarding.common.intro.style :as style]
+            [status-im2.common.resources :as resources]))
+
+(defonce carousel-index (reagent/atom 0))
```

Comment:
yeah sure, this will be refactored out anyway very soon because the carousel component will be handling it. This is just a temporary solution until that component and animations are implemented